### DISCUSSION TITLE
Help wanted: Example sentences for testing languages

### DISCUSSION THREAD

**Discussion by ines** [explosion/spaCy#1107] [Language Support]
When testing models and spaCy's performance in general, it's always nice to have a few text examples handy – especially for the languages you don't speak. We noticed this while testing the new Spanish and multi-language NER models. It's not always easy to find good, representative sentences, and copy-pasting whatever is on the Wikipedia front page that day isn't ideal either.

So here's the idea: For each language, we want to add **a simple list of text examples** that can be imported and processed – for example, to test if a model is working as expected. Ideally, those examples should include different types of sentences, grammatical constructions and named entities. The examples could also be used in spaCy's [test suite](https://github.com/explosion/spacy/tree/master/spacy/tests).

Examples could live with the language's data in `spacy.lang`.

```python
examples = [
    'Apple is looking at buying U.K. startup for $1 billion',
    'Autonomous cars shift insurance liability toward manufacturers',
    'San Francisco considers banning sidewalk delivery robots',
    'London is a big city in the United Kingdom.'
]
```

```python
from spacy.lang.en import examples

doc = nlp(examples[0]) # use one example
print([(t.text, t.pos_, t.tag_) for t in doc])

from spacy import displacy
docs = [nlp(doc) for doc in examples]
displacy.serve(*docs) # visualise all example sentences on one page
```

## ✅ Languages with models (8/8)

- [x] English `en` (can be covered by us)
- [x] German `de` (can be covered by us)
- [x] French `fr` (thanks @Gregory-Howard & @Tpt)
- [x] Spanish `es` (thanks @lucianosilvi)
- [x] Portuguese `pt` (thanks @oximer)
- [x] Italian `it` (thanks @MartinoMensio)
- [x] Dutch `nl` (thanks @redevries)
- [x] Greek `el` (thanks @Eleni170 & @giannisdaras)

## Languages with only tokenizers (29/40)

- [x] Albanian `sq` (thanks @AlbionaHoti)
- [x] Arabic `ar` (thanks @tzano)
- [x] Bengali `bn` (thanks @roshni-b)
- [x] Bulgarian `bg` (thanks @KristiyanVachev)
- [x] Catalan `ca` (thanks @mpuig) 
- [x] Chinese `zh` (thanks @renewang)
- [x] Croatian `hr` (thanks @isaric)
- [x] Danish `da` (thanks @mollerhoj)
- [x] Finnish `fi` (thanks @tjkemp)
- [x] Hebrew `he` (thanks @beneyal)
- [x] Hindi `hi` (thanks @weavermonkey)
- [x] Hungarian `hu` (thanks @oroszgy)
- [x] Indonesian `id` (thanks @geovedi)
- [x] Japanese `ja` (thanks @tokestermw)
- [x] Lithuanian `lt` (thanks @rokasramas)
- [x] Norwegian `nb` (thanks @luvogels)
- [x] Persian `fa` (thanks @azarezade)
- [x] Polish `pl` (thanks @adam-ra)
- [x] Romanian `ro` (thanks @janimo)
- [x] Russian `ru` (thanks @gsoul)
- [x] Sinhala `si` (thanks @keshan)
- [x] Swedish `sv` (thanks @RickardSjogren)
- [x] Tamil `ta` (thanks @Loghijiaha)
- [x] Tatar `tt` (thanks @aliiae)
- [x] Telugu `te` (thanks @sainathadapa)
- [x] Thai `th` (thanks @korakot & @wannaphongcom)
- [x] Turkish `tr` (thanks @DuyguA)
- [x] Ukrainian `uk` (thanks @juliamakogon)
- [x] Urdu `ur` (thanks @mirfan899)
- [ ] Afrikaans `af`
- [ ] Czech `cs`
- [ ] Icelandic `is`
- [ ] Irish `ga`
- [ ] Kannada `kn`
- [ ] Latvian `lv`
- [ ] Luxembourgish `lb`
- [ ] Serbian `sr`
- [ ] Slovak `sk`
- [ ] Slovenian `sl`
- [ ] Vietnamese `vi`

## "This is a sentence" example sentence (28/49)

If you happen to speak one of the tokenization-only languages above, a translation of `"This is a sentence."` would also be very helpful! The new [model docs](http://spacy.io/usage/models) have a "Quickstart" widget that shows the install command, plus an example sentence. If we ever add a model for one of the alpha languages, it'd be nice to have the example sentence available (and mistakes in the example would be pretty embarrassing, haha).

- [x] Albanian `sq`: "Kjo është një fjali." (thanks @AlbionaHoti)
- [x] Arabic `ar`:  هذه جملة (thanks @Ahmed0Sultan)
- [x] Bulgarian `bg`: "Това е изречение" (thanks @KristiyanVachev)
- [x] Danish `da`: "Dette er en sætning." (thanks @philiphoyos)
- [x] Dutch `nl`: "Dit is een zin." (thanks @redevries)
- [x] English `en`: "This is a sentence."
- [x] French `fr`: "C'est une phrase."
- [x] German `de`: "Dies ist ein Satz."
- [x] Greek `el`: "Αυτή είναι μια πρόταση." (thanks @Eleni170)
- [x] Hebrew `he`: "זהו משפט." (thanks @beneyal)
- [x] Hindi `hi`: "यह एक वाक्य है।" (thanks @weavermonkey & @psmehta21)
- [x] Hungarian `hu`: "Ez egy mondat." (thanks @oroszgy)
- [x] Indonesian `id`: "Ini adalah sebuah kalimat." (thanks @prdx)
- [x] Italian `it`: "Questa è una frase." (thanks @MartinoMensio)
- [x] Japanese `ja`: "これは文章です。" (thanks @tokestermw)
- [x] Norwegian `nb`: "Dette er en setning." (thanks @luvogels)
- [x] Polish `pt`: "To jest zdanie." (thanks @adam-ra)
- [x] Portuguese `pt`: "Esta é uma frase."
- [x] Romanian `ro`: "Aceasta este o propoziție" (thanks @janimo)
- [x] Russian `ru` : Это предложение. (thanks @buriy)
- [x] Sinhala `si`: "මෙය වාක්‍යයකි." (thanks @keshan)
- [x] Spanish `es`: "Esto es una frase."
- [x] Tamil `ta`: "இது ஒரு வாக்கியம்."  (thanks @arvindcheenu)
- [x] Telugu `te`: "ఇది ఒక వాక్యం." (thanks @sainathadapa)
- [x] Thai `th`: "นี่คือประโยค" (thanks @cvorasucha)
- [x] Turkish `tr`: "Bu bir cümledir." (thanks @DuyguA)
- [x] Ukrainian `uk`: "Це речення." (thanks @juliamakogon)
- [x] Urdu `ur`: "یہ ایک جملہ ہے۔" (thanks @mirfan899)
- [ ] Afrikaans `af`
- [ ] Bengali `bn`
- [ ] Catalan `ca`
- [ ] Chinese `zh`
- [ ] Croatian `hr`
- [ ] Czech `cs`
- [ ] Finnish `fi`
- [ ] Icelandic `is`
- [ ] Irish `ga`
- [ ] Kannada `kn`
- [ ] Latvian `lv`
- [ ] Lithuanian `lt`
- [ ] Luxembourgish `lb`
- [ ] Persian `fa`
- [ ] Serbian `sr`
- [ ] Slovak `sk`
- [ ] Slovenian `sl`
- [ ] Swedish `sv`
- [ ] Tatar `tt`
- [ ] Vietnamese `vi`

---

~We haven't fully decided how the text examples should be implemented~ The examples will live in a file `examples.py` in the language data – for example [`lang/en/examples.py`](https://github.com/explosion/spacy/tree/master/spacy/lang/en/examples.py). Currently, there's only a `sentences` list, but more examples can be added in the future.

If you want to help out and contribute a few examples, you can submit a PR, or simply post them here! 🙂

**Comment by Gregory-Howard** [comment]
In french : 
Translations of examples given before.
```
examples = [
"Apple cherche a acheter une startup anglaise pour 1 milliard de dollard",
"Les voitures autonomes voient leur assurances décalées vers les constructeurs",
"San Francisco envisage d'interdire les robots coursiers",
"Londres est une grande ville du Royaume-Uni"
]
```
Title of news :
```
examples = [
"L’Italie choisit ArcelorMittal pour reprendre la plus grande aciérie d’Europe",
"Apple lance HomePod parce qu'il se sent menacé par l'Echo d'Amazon",
"La France ne devrait pas manquer d'électricité cet été, même en cas de canicule",
"Nouvelles attaques de Trump contre le maire de Londres"
]
```

**Comment by Tpt** [comment]
Some questions in English and French:

en_examples = [
"Where are you?",
"Who is the president of France?",
"Where is the capital of the United States?",
"When is Barack Obama born?"
]

fr_examples = [
"Où es-tu ?",
"Qui est le président de la France ?",
"Où est la capitale des Etats-Unis ?",
"Quand est né Barack Obama ?"
]

**Comment by oroszgy** [comment]
A related idea: you might want to test these strings as well to check that none of them causes a problem: https://github.com/minimaxir/big-list-of-naughty-strings

**Comment by ines** [comment]
Thanks for the examples so far! 👍

@oroszgy Thanks, that's a good idea for the tokenizer tests, especially the crazy unicode stuff! I'm actually pretty curious to see if spaCy can handle all of them – I'll start adding a test with a selection of them on `develop`. (In v2.0 alpha, there are currently still some issues in the tokenizer serialization across Python versions, which we suspect might be unicode-related. So this might be pretty helpful for debugging this issue as well.)

**Comment by unography** [comment]
@ines Hi, how could one contribute for the Indian languages, like Hindi and Bengali?

**Comment by ines** [comment]
@eldor4do There's no Hindi tokenization support yet (although it looks like some people have started working on it, see #641). 

Bengali tokenization is already supported – so a few sentences in Bengali would be great 👍  [Here's](https://github.com/explosion/spaCy/blob/master/spacy/tests/bn/test_tokenizer.py) what we have in the tokenizer tests so far. Just had a look at some of the test examples in Google Translate and they seem pretty good – so more examples like that would be nice.

**Comment by lucianosilvi** [comment]
For **Spanish**:
```
es_examples = [
    'Apple está buscando comprar una startup del Reino Unido por mil millones de dólares',
    'Los coches autónomos delegan la responsabilidad del seguro en sus fabricantes',
    'San Francisco analiza prohibir los robots delivery',
    'Londres es una gran ciudad del Reino Unido',
    'El gato come pescado',
    'Veo al hombre con el telescopio',
    'La araña come moscas',
    'El pingüino incuba en su nido'
]
```

**Comment by adam-ra** [comment]
For Polish:
```
pl_examples = [
    'Poczuł przyjemną woń mocnej kawy.',
    'Istnieje wiele dróg oddziaływania substancji psychoaktywnej na układ nerwowy.',
    'Powitał mnie biało-czarny kot, płosząc siedzące na płocie trzy dorodne dudki.',
    'Nowy abonament pod lupą Komisji Europejskiej',
    'Czy w ciągu ostatnich 48 godzin spożyłeś leki zawierające paracetamol?',
    'Kto ma ochotę zapoznać się z innymi niż w książkach przygodami Muminków i ich przyjaciół, temu polecam komiks Tove Jansson „Muminki i morze”.'
]
```

**Comment by beneyal** [comment]
Hebrew examples:

```python
he_this_is_a_sentence = 'זהו משפט.'

he_examples_news = [
    'סין מקימה קרן של 440 מיליון דולר להשקעה בהייטק בישראל',
    'רה"מ הודיע כי יחרים טקס בחסותו',
    'הכנסת צפויה לאשר איכון אוטומטי של שיחות למוקד 100',
    'תוכנית לאומית תהפוך את ישראל למעצמה דיגיטלית'
]

he_examples_misc = [
    'סע לשלום, המפתחות בפנים.',
    'מלצר, פעמיים טורקי!',
    'ואהבת לרעך כמוך.',
    'היום נעשה משהו בלתי נשכח.'
]

he_examples_questions = [
    'איפה הילד?',
    'מיהו נשיא צרפת?',
    'מהי בירת ארצות הברית?',
    "איך קוראים בעברית לצ'ופצ'יק של הקומקום?",
    'מה הייתה הדקה?',
    'מי אומר שלום ראשון, זה שעולה או זה שיורד?',
]

**Comment by RickardSjogren** [comment]
For Swedish:

```
examples = [
    'Apple överväger att köpa brittisk startup för 1 miljard dollar.',
    'Självkörande bilar förskjuter försäkringsansvar mot tillverkare.',
    'San Fransisco överväger förbud mot leveransrobotar på trottoarer.'.
    'London är en storstad i Storbritannien.'
]
```

**Comment by adam-ra** [comment]
“This is a sentence.” -> „To jest zdanie.” (Polish)

**Comment by oximer** [comment]
For Portuguese BR

pt_examples = [
    'Apple está querendo comprar uma startup do Reino Unido por 100 milhões de dólares',
    'Carros autônomos empurram a responsabilidade do seguro para os fabricantes.'
    'São Francisco considera banir os robôs de entrega que andam pelas calçadas',
    'Londres é a maior cidade do Reino Unido'
]

**Comment by mollerhoj** [comment]
For Danish:
```
examples = [
    'Apple overvejer at købe et britisk statup for 1 milliard dollar',
    'Selvkørende biler flytter forsikringsansvaret over på producenterne',
    'San Francisco overvejer at forbyde leverandørrobotter på fortov',
    'London er en stor by i Storbritannien'
]
```

**Comment by luvogels** [comment]
For Norwegian NB:

```
examples = [
    'Apple vurderer å kjøpe britisk oppstartfirma for en milliard dollar',
    'Selvkjørende biler flytter forsikringsansvaret over på produsentene ',
    'San Francisco vurderer å forby robotbud på fortauene',
    'London er en stor by i Storbritannia.'
]
```
“This is a sentence.” -> "Dette er en setning."

**Comment by MartinoMensio** [comment]
For Italian:

```
examples = [
    'Apple vuole comprare una startup del Regno Unito per un miliardo di dollari',
    'Le automobili a guida autonoma spostano la responsabilità assicurativa verso i produttori',
    'San Francisco prevede di bandire i robot di consegna porta a porta',
    'Londra è una grande città del Regno Unito.'
]
```

**Edit:** "This is a sentence" --> "Questa è una frase"

**Comment by oroszgy** [comment]
@ines The Hungarian translation of `"This is a sentence."` is `"Ez egy mondat."` If needed, you can use these sentences for testing:
```
examples = [
    'Az Apple egy brit startup vásárlását tervezi 1 milliárd dollár értékben.',
    'San Francisco vezetése mérlegeli a járdát használó szállító robotok betiltását.',
    'London az Egyesült Királyság egy nagy városa.'
]
```

**Comment by renewang** [comment]
For traditional Chinese/zh_tw (follow the same example above):
examples = [
    蘋果公司正考量用一億元買下英國的新創公司,
    自駕車將保險責任歸屬轉移至製造商,
    舊金山考慮禁止送貨機器人在人行道上行駛,
    倫敦是英國的大城市
    'Apple is looking at buying U.K. startup for $1 billion',
    'Autonomous cars shift insurance liability toward manufacturers',
    'San Francisco considers banning sidewalk delivery robots',
    'London is a big city in the United Kingdom.'
]
No idea what simplified Chinese / zh_cn could be. Hope this example help!

**Comment by tokestermw** [comment]
Japanese

```
examples = [
    'アップルがイギリスの新興企業を１０億ドルで購入を検討',
    '自動運転車の損害賠償責任、自動車メーカーに一定の負担を求める',
    '歩道を走る自動配達ロボ、サンフランシスコ市が走行禁止を検討',
    'ロンドンはイギリスの大都市です。',
]
```

`'This is a sentence'` -> `'これは文章です。'`

**Comment by redevries** [comment]
Dutch

This is a sentence --> Dit is een zin

```
examples = [
    'Apple overweegt om voor 1 miljard een U.K. startup te kopen'
    'Autonome auto's verschuiven de verzekeringverantwoordelijkheid naar producenten'
    'San Francisco overweegt robots op voetpaden te verbieden'
    'Londen is een grote stad in het Verenigd Koninkrijk' 
]
```

**Comment by jmlange** [comment]
@Gregory-Howard a suggested improvement to the French translations (changes highlighted between underscores)

```
> examples = [
> "Apple cherche __à__ acheter une startup anglaise pour 1 milliard de dollar__s__",
> "Les voitures autonomes __déplacent la responsabilité de l'assurance__ vers les constructeurs",
> "San Francisco envisage d'interdire les robots coursiers __sur les trottoirs__",
> "Londres est une grande ville du Royaume-Uni"
> ]
```

**Comment by ines** [comment]
@jmlange Thanks – only saw this now, sorry! If you have time, would you mind submitting this as a pull request? This will make it a little easier for us to see the exact changes!

**Comment by gsoul** [comment]
examples = [
    'Apple is looking at buying U.K. startup for $1 billion',
    'Autonomous cars shift insurance liability toward manufacturers',
    'San Francisco considers banning sidewalk delivery robots',
    'London is a big city in the United Kingdom.'
]

Russian:

examples = [
    'Apple рассматривает возможность покупки стартапа из Соединенного Королевства за $1 млрд',
    'С появлением беспилотных автомобилей ответственность за страховые случаи переносится на производителя',
    'В Сан Франциско рассматривается возможность запрета роботов-курьеров, которые перемещаются по тротуару',
    'Лондон - большой город Соединенного Королевства'
]

Ukrainian:

examples = [
    'Apple розглядає можливість придбання стартапу з Великобританії за $1 мільярд доларів',
    'З появою безпілотних автомобілів відповідальність за страхові випадки переноситься на виробника',
    'У Сан Франциско розглядається можливість заборони роботів-кур\'єрів, що їздять тротуарами',
    'Лондон - велике місто Сполученого Королівства'
]

**Comment by ines** [comment]
@gsoul Thanks a lot! We don't have a `Ukrainian` language class yet, but it'd be cool to support it in the future. 

So if you – or someone else who comes across this in the future – wants to take this on, this would be amazing. Here's the [Russian language data](https://github.com/explosion/spaCy/tree/master/spacy/lang/ru) and the [adding languages docs](https://spacy.io/usage/adding-languages) for inspiration. I think the [`pymorphy2`](https://github.com/kmike/pymorphy2) package which is currently used by the Russian lemmatizer also supports Ukrainian.

**Comment by azarezade** [comment]
Persian translation of 
```
examples = [
    'Apple is looking at buying U.K. startup for $1 billion',
    'Autonomous cars shift insurance liability toward manufacturers',
    'San Francisco considers banning sidewalk delivery robots',
    'London is a big city in the United Kingdom.'
]
```
are
```
examples = [
    'اپل به دنبال خرید یک استارتاپ انگلیسی به قمیت ۱ میلیارد دلار است',
    'ماشین‌های خودکار مسئولیت بیمه نسبت به تولیدکنندگان را تغییر می‌دهند',
    'سان فرانسیسکو در نظر دارد ربات‌های تحویل‌دهنده در پیاده‌رو را ممنوع کند',
    'لندن شهر بزرگی در انگلستان است'
]
```
Also `This is a sentence` translates to `این یک جمله است`.

**Comment by wannaphong** [comment]
For Thai:

```python
examples = [
    'แอปเปิลกำลังพิจารณาจะซื้อบริษัทสตาร์ทอัพของสหราชอาณาจักรในราคา 1 พันล้านเหรียญ',
    'รถยนต์อิสระผลักภาระความรับผิดทางประกันภัยไปยังฝ่ายผู้ผลิตรถยนต์',
    'เมืองซานฟรานซิสโกกำลังพิจารณาจะห้ามหุ่นยนต์รับส่งของบนทางเท้า',
    'ลอนดอนเป็นเมืองขนาดใหญ่ในสหราชอาณาจักร'
]
```

Thank you @korakot (https://www.facebook.com/groups/thainlp/permalink/519628468418648/)

**Comment by DuyguA** [comment]
Turkish:

This is a sentence  ->  Bu bir cümledir.
Where are you -> Neredesin?  2nd person singular
Where are you? -> Neredesiniz? 2nd person plural

```
sentences = [
    "Neredesin?",
    "Neredesiniz?",
    "Bu bir cümledir.",
    "Sürücüsüz araçlar sigorta yükümlülüğünü üreticilere kaydırıyor.",
    "San Francisco kaldırımda kurye robotları yasaklayabilir."
    "Londra İngiltere'nin başkentidir.",
    "Türkiye'nin başkenti neresi?",
    "Bakanlar Kurulu 180 günlük eylem planını açıkladı.",
    "Merkez Bankası, beklentiler doğrultusunda faizlerde değişikliğe gitmedi."
]
```

I'll make a pull request soon.

**Comment by weavermonkey** [comment]
Hindi:

```
sentences = [
    "एप्पल एक यूक स्टार्टअप को १ अरब डॉलर के लिए खरीदने का सोच रहा है|",
    "गाड़ियां अब बीमा दायित्व को अब उत्पादक के ऊपर दाल रहे हैं|",
    "सं फ्रांसिस्को फुटपाथ वितरण रोबोट पर प्रतिबंध लगाने का विचार कर रहा है|",
    "लन्दन यूके में एक बड़ा शहर है|",
    "कहाँ हो?",
    "फ्रांस का राष्ट्रपति कौन है?"
    "अमेरिका का राजधानी क्या है?",
    "बराक ओबामा का जनम कब हुआ?"
    ]
```
`hindi_this_is_sentence = "य हक वाकया है|"`

**Comment by bwanglzu** [comment]
For simplified Chinese:

```python
sentences = [
'苹果公司正考虑用一亿元购买英国的创新公司',
'自动驾驶汽车将保险责任转移至制造商',
'旧金山考虑禁止送货机器人在人行道上行驶',
'伦敦是英国的大城市'
]
```

`This is a sentence` -> `这是一个句子`  (zh-cn)
`This is a sentence` -> `這是一個句子`  (zh-tw)

Basically Chinese is now separated into two branches: traditional Chinese (zh-tw) mainly used in Taiwan, HongKong, Macau & Overseas Chinese community.

Simplified Chinese (zh-cn) mainly used in mainland China (PRC).

Thanks for @renewang for providing zh-tw, greeting :)

**Comment by cvorasucha** [comment]
For Thai:
```
This is a sentence -> นี่คือประโยค
```

**Comment by mirfan899** [comment]
English:
`This is a sentence.`
For Urdu:
`یہ ایک جملہ ہے۔`

**Comment by mirfan899** [comment]
English sentences:

```python
examples = [
"London is a big city in the United Kingdom.",
"Where are you?",
"Who is the president of France?",
"What is the capital of the United States?",
"When was Barack Obama born?"
]
```
Translated to Urdu
```python
examples = [
"لندن یونائیٹڈ کنگڈم میں ایک بڑا شہر ہے۔",
"تم کہاں ہو؟",
"فرانس کا صدر کون ہے؟",
"یونائیٹڈ سٹیٹس کا دارالحکومت کیا ہے؟",
"براک اباما کب پیدا ہوے؟"
]

**Comment by janimo** [comment]
Romanian: "Aceasta este o propoziție"

**Comment by psmehta21** [comment]
@weavermonkey’s translation for “This is a sentence” is incorrect. It should be यह एक वाक्य है।

**Comment by philiphoyos** [comment]
“This is a sentence.” -> "Dette er en sætning." (Danish)

**Comment by Eleni170** [comment]
"This is a sentence." -> "Αυτή είναι μια πρόταση." (Greek)

**Comment by sainathadapa** [comment]
"This is a sentence." -> "ఇది ఒక వాక్యం." (Telugu (te))

**Comment by khudhairJassim** [comment]
Arabic
examples = [
     "تتطلع شركة أبل إلى شراء شركة ناشئة في المملكة المتحدة مقابل مليار دولار" ،
     "السيارات الخصوصي تحول مسؤولية التأمين نحو المصنعين" ،
     " سان فرانسيسكو تاخذ بعين الاعتبار حظر روبوتات التوصيل على رصيف المشاة" ،
     ".لندن هي مدينة كبيرة في المملكة المتحدة" 
                     ]

**Comment by loghijiaha** [comment]
I would like to support to spaCy by adding Tamil language .@ines

**Comment by mpuig** [comment]
For Catalan:

```
This is a sentence -> Això és una frase
```

**Comment by juliamakogon** [comment]
Ukrainian:
Це речення.

нд, 17 лют. 2019 о 17:41 Marc Puig <notifications@github.com> пише:

**Comment by VariableVasasMT** [comment]
> Hindi:
> 
> ```
> sentences = [
>     "एप्पल एक यूक स्टार्टअप को १ अरब डॉलर के लिए खरीदने का सोच रहा है|",
"एप्पल एक यूके की स्टार्टअप को १ अरब डॉलर मे खरीदने का सोच रहा है|"
>     "गाड़ियां अब बीमा दायित्व को अब उत्पादक के ऊपर दाल रहे हैं|",
should be स्वचालित गाड़ियां अब बीमा के दायित्व को अब उत्पादक के ऊपर डाल रही हैं|
>     "सं फ्रांसिस्को फुटपाथ वितरण रोबोट पर प्रतिबंध लगाने का विचार कर रहा है|",
>     "लन्दन यूके में एक बड़ा शहर है|",
>     "कहाँ हो?",
>     "फ्रांस का राष्ट्रपति कौन है?"
>     "अमेरिका का राजधानी क्या है?",
"अमेरिका की राजधानी क्या है?"
>     "बराक ओबामा का जनम कब हुआ?"
>     ]
> ```
> 
> `hindi_this_is_sentence = "य हक वाकया है|"`
यह एक वाक्य है|

**Comment by prdx** [comment]
I saw Indonesian model is in progress, here is the Indonesian translation:

```
This is a sentence -> Ini adalah sebuah kalimat
```

**Comment by AlbionaHoti** [comment]
Albanian:

 ``` 
examples = [
    'Apple po shqyrton blerjen e nje shoqërie të U.K. për 1 miliard dollarë',
    'Makinat autonome ndryshojnë përgjegjësinë e sigurimit ndaj prodhuesve',
    'San Francisko konsideron ndalimin e robotëve të shpërndarjes',
    'Londra është një qytet i madh në Mbretërinë e Bashkuar.' 
]

 This is a sentence -> Kjo është një fjali 
```

**Comment by KristiyanVachev** [comment]
For **Bulgarian**:

`This is a sentence -> Това е изречение
`

```
examples = [
    'Епъл иска да купи английски стартъп за 1 милиард долара.' 
    'Автономните коли прехвърлят застрахователната отговорност към производителите.' 
    'Сан Франциско обмисля забрана на роботи доставящи по тротоари. ' 
    'Лондон е голям град в Обединеното Кралство.'
]
```

There are some specifics about the use of Bulgarian.

- In _'Apple is looking at buying U.K. startup for $1 billion'_, we don't have a word for _UK_ and we use the equivalent of _"English startup"_. It may be more accurate to say _"великобритански стартъп"_ - _"great-britaneese startup"_, but I have never seen it being used in practice.
- In _'San Francisco considers banning sidewalk delivery robots',_ we don't structure phrases like _"sidewalk delivery robots"_ but more like _"robots that deliver on sidewalks"_. 

Spacy is great and I would like to contribute more for the Bulgarian pack in the future.

**Comment by ship-it-mind** [comment]
"This is a sentence" example sentence in **Arabic**

```
This is a sentence  -> هذه جملة
```

**Comment by mzeidhassan** [comment]
Do you need further help with Arabic, @ines ? More examples, translation review, etc.?

**Comment by buriy** [comment]
>"This is a sentence." example sentence 

Russian: 
Это предложение.

**Comment by arvindcheenu** [comment]
Translated the examples below into Tamil:
```
'Apple is looking at buying U.K. startup for $1 billion',
'Autonomous cars shift insurance liability toward manufacturers',
'San Francisco considers banning sidewalk delivery robots',
'London is a big city in the United Kingdom.'
```
translates to
```
"இது ஒரு வாக்கியம்.",
"ஆப்பிள் நிறுவனம் யு.கே. தொடக்க நிறுவனத்தை ஒரு லட்சம் கோடிக்கு வாங்கப் பார்க்கிறது",
"தன்னாட்சி கார்கள் காப்பீட்டு பொறுப்பை உற்பத்தியாளரிடம் மாற்றுகின்றன",
"நடைபாதை விநியோக ரோபோக்களை தடை செய்வதை சான் பிரான்சிஸ்கோ கருதுகிறது",
"லண்டன் ஐக்கிய இராச்சியத்தில் ஒரு பெரிய நகரம்."
```

**Comment by rasyidf** [comment]
There is something wrong with the Indonesian Example sentences, it contains sensitive phrases as **terorism** and **bombing**, while there are tons of examples available.

sending a pull request for it.

**Comment by jairwen** [comment]
Simplified Chinese / zh_cn


> examples = [
Apple is looking at buying U.K. startup for $1 billion',
'Autonomous cars shift insurance liability toward manufacturers',
'San Francisco considers banning sidewalk delivery robots',
'London is a big city in the United Kingdom.'
]

->

>examples = [
苹果公司正考虑用1亿美元收购英国初创公司',
'自动驾驶汽车使得保险责任转移至生产厂家',
'旧金山考虑禁止人行道式送餐机器人',
'伦敦是英国的一个大城市'
]


>This is a sentence ->  这是一个句子。

**Comment by elisno** [comment]
Icelandic examples:

```python
examples = [
    'Apple íhugar að kaupa breskt sprotafyrirtæki fyrir 1 milljarð dollara',
    'Sjálfstýrðir bílar færa tryggingarábyrgð yfir á framleiðendur',
    'San Francisco íhugar að banna afhendingarvélmenni á gangstéttum',
    'London er stór borg í Bretlandi.'
]
```

Note: Similarly to the Bulgarian translations (as with other translations that I see here), `U.K. startup` would turn into `British startup` when translating English -> Icelandic -> English. 

`afhendingarvélmenni` is just some neologism for `delivery robots`, which I suspect that current automatic translations would opt for.

As mentioned for other languages, the translation for `sidewalk delivery robots` would be structured as `robots on sidewalks that do deliveries` (if it were back-translated).
Perhaps one could go overboard coin an even longer word by translating `sidewalk delivery robot` as `gangstéttarafgreiðsluvélmenni`, but I'm afraid it won't claim the record as the [longest Icelandic word](https://icelandmonitor.mbl.is/news/culture_and_living/2018/10/15/meet_the_longest_word_in_icelandic/). 🤪

--- 

Icelandic "This is a sentence" example sentence:

```python
Þetta er setning
```
